packages/frontend/src/components/list-view/ListView.tsx

```diff
@@ -0,0 +1,45 @@
+import { ClayPaginationBarWithBasicItems } from '@clayui/pagination-bar';
+import React from 'react';
+
+import spritemap from '@/assets/spritemap.svg';
```

Comment:
The project already have this spritemap in _app.tsx, we could remove it from here, soon!!